KD1OUR Task 1

conTroll opened the pull request:
1. feladat megoldása
   Ismert nyelv: Java
   Új nyelv: Scala

conTroll committed:
First commit: README

conTroll committed:
Update README.md

conTroll committed:
Task1: State Machine, Java Solution

conTroll committed:
Merge branch 'master' of github.com:conTroll/langs-and-paradigms

conTroll committed:
JDK character mapping error javítva

conTroll committed:
Update README.md

conTroll committed:
Update README.md

conTroll committed:
- Move Java project to separate dir
- Task1: Solution in Scala

conTroll committed:
Merge branch 'master' of github.com:conTroll/langs-and-paradigms

conTroll committed:
Update README.md

olikasg commented:
A peldak nagyon szepen ki vannak dolgozva. Hianyzik a README.md-bol egy osszefoglalas, hogy melyik nyelven mit talaltal konnyunek, neheznek. Kicsit ertekelni kellene a megoldasokat. Ha ezt hozzairod, akkor elfogadjuk a megoldast.

conTroll committed:
- README.md fix: fordítás Scala-n javítva
- README.md: tapasztalatok, különbségek a két nyelv között
- a kommentek és leírások magyarra lettek fordítva mindenütt egységesen

conTroll commented:
Pótoltam. :)

conTroll committed:
Update README.md

olikasg commented:
Szep munka! Elfogadva.
